server/src/main/java/no/ntnu/klubbhuset/service/AdminOrganizationService.java

```diff
@@ -75,4 +76,30 @@ public boolean isAdminOfOrganization(Long organizationId) {
 
         return result;
     }
+
+    @PermitAll
+    public Response harMemberPaid(Long organizationId, User user) {
```

Comment:
"har"memberPaid

Should we really mix Norwegian and English in our naming scheme?

Reply:
Burde vel heller ikke være permitAll

Reply:
done